CONTRIBUTING.md

```diff
@@ -221,12 +221,12 @@ logical units of work. You are free to propose a patch with one or many
 commits, as long as their atomicity is respected. This means that no unrelated
 changes should be included in a commit.
 
-For example: you are writing a patch to fix a bug, but in your endeavour, you
+For example: you are writing a patch to fix a bug, but in your endeavor, you
```

Comment:
american vs english spelling. I'm not sure what our preferred style here is; I'd err on the side of leaving as-is.